test/test_helper.rb

```diff
@@ -17,8 +17,9 @@
 require 'rails-controller-testing'
 Rails::Controller::Testing.install
 
-def request_params(params)
-  { params: params }
+# FIXME: update tests now to inline this
+def request_params(params, **options)
+  { params: params }.merge(options)
```

Comment:
Is this still needed now that we dropped Rails 4 support?

Reply:
No it's just temporary. I still need to inline this in all test which I will do I just didn't because of #466. I can do that now though, it's no problem. What do you think is best? Should I do that update now here or merge this in as is, then we merge #466 and I submit a new PR to remove `request_params` in all our tests?

Reply:
I see. I mentioned Rails 4 because it was mentioned in the commit message and it seemed no longer relevant.
 
> What do you think is best? Should I do that update now here or merge this in as is, then we merge #466 and I submit a new PR to replace request_params in all our tests and inline the params locally?

I wasn't sure either, so I ended up inlining it myself in https://github.com/activeadmin/inherited_resources/pull/467/commits/8c50ee714ecb91e098a2ebf8251d35355aae435a :smiley:

Reply:
I think its good now, only thing is that the diff has a mix of hash key styles that I find a bit distracting. Can be fixed, but no big deal.

Reply:
Ok great thanks! 👍🏻 I'll get this in now.